feat: Init

sandropark opened the pull request:
- 기본 프로젝트 구조입니다.
- 이전 PR #1 닫고 다시 만들었습니다.

sandropark committed:
feat: Init

hrkim3468 commented on src/test/kotlin/com/sandro/realtime/RealtimeSandroApplicationTests.kt:
```diff
@@ -0,0 +1,15 @@
+@SpringBootTest
+class RealtimeSandroApplicationTests {
+
+    @Test
```
테스트가 좀더 보강되면 좋을것 같아요.

sandropark replied:
보강하겠습니다!

hrkim3468 commented:
LGTM

sandropark committed:
feat: Pull Request용 GitHub Actions CI 워크플로우 추가

PR 이벤트(opened, synchronize, reopened)에서 실행되는 CI 워크플로우 추가.
Amazon Corretto Java 21과 Gradle 캐싱을 사용하여 프로젝트 빌드 수행.

sandropark committed:
feat: PR 워크플로우에 ktlint 코드 품질 검사 통합

  - build.gradle에 ktlint 플러그인 추가 (v12.1.1)
  - 코드 스타일 자동 포맷팅 적용 (탭 → 스페이스, 불필요한 공백 제거)
